### DISCUSSION TITLE
Github Skills Broken

### DISCUSSION THREAD

**Discussion by stevelk** [community/community#58602] [Other Feature Feedback, Questions, & Ideas]
### Hi everyone,

Wanted to see what novice information regarding how to use github was available. Had existing (hardly used login (_stevelk_).  Did the following using my Chrome Linux browser:

1. went to https://docs.github.com/en/get-started/quickstart/git-and-github-learning-resources
2. Clicked on "Github Skills", went to https://skills.github.com/
3. Pressed the "Start with introduction to Github" button. Went to https://github.com/skills/introduction-to-github. So far so good.
4. Read Introduction/Welcome/How to start this course sections. 
5. Followed directions under "How to start this course".
6. Create a New repository  screen appeared.
7. Pressed the "Create Repository" button using defaults.
8. Created new repository!
9. Unfortunately, I was sent back to the intro to the SAME github template, with the SAME INFORMATION  on the page. I thought I might have made a mistake, and started over at step 6 above. I ended up creating another repository before confirming I was not moved forward with the lesson..
10. I ended up clicking on the cat, and saw the repositories, with the next set of information.
11. ugh. deleted the second repository. Please fix this. Tell them to click on the cat or something. frustrating for regular developer folks..

Thanks.

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @stevelk , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
